asam-ev/qc-opendrive
# CCB Review: Prepare v1.0.0-rc.1 release candidate

**Description**

This PR prepares for the release candidate 1.0.0rc1. After this PR is merged into `main`, the tag `v1.0.0-rc1` will be created and the Python package `asam-qc-opendrive 1.0.0rc1` will be published to PyPi.

**Main changes**

1. Add release disclaimer
2. Remove the optional argument `build_date` when registering checker bundle
3. Add documentation about how to implement new checkers
4. Add quotation to example windows manifest to handle spaces (https://github.com/asam-ev/qc-framework/issues/189)
5. Use `asam-qc-baselib` from the official PyPi server with the version range `^1.0.0rc1`
6. Use `typing.Optional` instead of `Union[None, ...]`.
7. Fix an issue in the new lane appear rule as described in #112 

**How was the PR tested?**

1. Tests are documented in each PR below.
2. Unit tests.

**Notes**

A test release in the Test PyPi server is available [here](https://test.pypi.org/project/asam-qc-opendrive/1.0.0rc1/).
